.github/workflows/osd-increment-plugin-versions.yml

```diff
@@ -46,10 +46,8 @@ jobs:
           - '1.3'
           - 2.x
           - main
-          - '2.15'
-          - '2.16'
-          - '2.17'
```

Comment:
I think we still want to keep these for patch increments like https://github.com/opensearch-project/k-NN/pull/1944 (See the conversation there)

Reply:
Would still recommend to keep all versions @prudhvigodithi can add too. Its just keeping the automation running from our end. At the end its on maintainers to accept it or not.